src/services/discord/info.js

```diff
@@ -0,0 +1,91 @@
+const { version } = require('../../../package.json')
+const botconfig = require('../../../botconfig.json')
```

Comment:
Do these require statements work? I don't have the repo pulled but it looks like they may be one too many levels deep, and `botconfig.json` looks like it was removed

Reply:
Yes they both work fine, botconfig.json is .gitignore'd because it contains the bot token